src/comps/pages/admin/AdminMeetingCard.js

```diff
@@ -0,0 +1,140 @@
+import { useState } from "react";
+import Card from "@mui/material/Card";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemAvatar from "@mui/material/ListItemAvatar";
+import Avatar from "@mui/material/Avatar";
+import ListItemText from "@mui/material/ListItemText";
+import moment from "moment";
+
+import { useHistory } from "react-router-dom";
+import { Typography } from "@mui/material";
+import smartTimespan from "../../../utils/smartTimespan";
+import { triggerMeetingDialog } from "../../meetings/MeetingPreviewDialog";
+import Button from "@mui/material/Button";
+import Box from "@mui/material/Box";
+import Alert from "../../ui/Alert";
+
+const classes = {
+	card: {
+		position: "relative"
+	},
+	meetingContent: {
+		padding: "0 1rem",
+		overflow: "auto"
+	},
+	orgHeading: {
+		cursor: "pointer",
+		"&:hover": {
+			opacity: 0.9
+		}
+	},
+	descriptionContainer: {
+		overflowWrap: "anywhere"
+	},
+	title: {
+		fontWeight: 700
+	},
+	buttonContainer: {
+		width: "100%",
+		height: "50px",
+		bottom: 0
+	}
+};
+/* Meeting Card to let admin manage meeting */
+const AdminMeetingCard = ({
```

Comment:
Normally I would encourage reuse of the normal MeetingCard but hey if it works it works; will deploy to the beta domain for testing